### ISSUE TITLE
template for official embed code

### ISSUE THREAD

**Issue by martinvirtel** [dpa-newslab/liveblog-embed-configurator#2]
```html
<iframe src="<% URL %>" data-szm-st="<% szm-st %>" data-szm-cp="<% foo-szm-cp %>" data-szm-co="<% foo-szm-co %>" data-ga-property="<% UA-foo%>" onload="var lb_frame = this; this.contentWindow.postMessage({type: 'analytics', payload: JSON.stringify(lb_frame.dataset)}, lb_frame.src);" scrolling=no frameborder=0 width="320px" height="800" style="max-width: 320px; overflow: hidden; border:0"><p>Ihr Browser unterstützt keine iframes. Bitte verfolgen Sie die dpa-Live-Berichterstattung <a href="<% URL %>unter diesem Link</a>.</p></iframe>
```

**Comment by cornelia1** [comment]
Soll ich das einbauen?

**Comment by martinvirtel** [comment]
ich habe noch eine Rückfrage bei Paul laufen. 320 Pixel Breite kann nicht
richtig sein.

2016-06-29 17:30 GMT+02:00 cornelia1 <notifications@github.com>:

> Soll ich das einbauen?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/dpa-newslab/liveblog-embed-configurator/issues/2#issuecomment-229393655>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AJC91tWefKXRcVM4o90F82fcVZm4VTohks5qQo-VgaJpZM4JANsP>
> .
>



-- 

Martin Virtel <mvirtel@dpa-newslab.com>
+49 172 9921396

<https://market.scribblelive.com/Events/Client/10544>

**Comment by psnewslab** [comment]
Klar, der ist alt. Sollen width/height nicht ohnehin parametrisiert werden?
Ich weiß nur, dass ich persönlich 768 px Breite ideal finde, aber das ist ja irrelevant an dieser Stelle.
Nehmt doch den Embedcode ausm Theme-Repository mit den alten Scribble-Werten.

```
<iframe src="//" data-szm-st="" data-szm-cp="" data-szm-co="" data-ga-property="" onload="var lb_frame = this; this.contentWindow.postMessage({type: 'analytics', payload: JSON.stringify(lb_frame.dataset)}, lb_frame.src);" scrolling=no frameborder=0 width="500" height="800" style="overflow: hidden; border:0"><p>Ihr Browser unterstützt keine iframes.</p></iframe>
```

Die width ist im Ggs zu Scribble aus Theme-Sicht egal, weil die timeline eine max-width hat.
Nur eine gewisse Mindesthöhe ist natürlich schön.

**Comment by martinvirtel** [comment]
Ja, parametrisiert. Als Theme-Autor darfst Du Dir aber den default wünschen
;-)
Vorschläge also:

Breite: Mindestens 320, Wert: 100%, nicht parametrisierbar.

Höhe: Parametrisierbar, Standard 600px

Am 30. Juni 2016 um 11:09 schrieb psnewslab <notifications@github.com>:

> Klar, der ist alt. Sollen width/height nicht ohnehin parametrisiert werden?
> Ich weiß nur, dass ich persönlich 768 px Breite ideal finde, aber das ist
> ja irrelevant an dieser Stelle.
> Nehmt doch den Embedcode ausm Theme-Repository mit den alten
> Scribble-Werten.
>
> <iframe src="//" data-szm-st="" data-szm-cp="" data-szm-co="" data-ga-property="" onload="var lb_frame = this; this.contentWindow.postMessage({type: 'analytics', payload: JSON.stringify(lb_frame.dataset)}, lb_frame.src);" scrolling=no frameborder=0 width="500" height="800" style="overflow: hidden; border:0"><p>Ihr Browser unterstützt keine iframes.</p></iframe>
>
> Die width ist im Ggs zu Scribble aus Theme-Sicht egal, weil die timeline
> eine max-width hat.
> Nur eine gewisse Mindesthöhe ist natürlich schön.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/dpa-newslab/liveblog-embed-configurator/issues/2#issuecomment-229604816>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AJC91vgeJ2oZ9Es6cq-7YZ3cZe1RWIDlks5qQ4fXgaJpZM4JANsP>
> .
>



-- 

Martin Virtel <mvirtel@dpa-newslab.com>
+49 172 9921396

<https://market.scribblelive.com/Events/Client/10544>

**Comment by psnewslab** [comment]
ok, siehe https://github.com/dpa-newslab/liveblog-theme-dpa/blob/master/embedcode.html

```
<iframe src="" data-szm-st="" data-szm-cp="" data-szm-co="" data-ga-property="" onload="var lb_frame = this; this.contentWindow.postMessage({type: 'analytics', payload: JSON.stringify(lb_frame.dataset)}, lb_frame.src);" scrolling=no frameborder=0 width="100%" height="600" style="overflow: hidden; border: 0; min-width: 320px;"><p>Ihr Browser unterstützt keine iframes.</p></iframe>
```

**Comment by psnewslab** [comment]
sorry, `scrolling=no` soll für Kunden natürlich raus.

**Comment by martinvirtel** [comment]
Hier steht der Code jetzt immer offiziell:

https://github.com/dpa-newslab/liveblog-theme-dpa/blob/master/embedcode.html


Kannst Du bitte einbauen, @cornelia1 ? Merci

**Comment by cornelia1** [comment]
ich habe diesen iframe ohne 
_<iframe src="" data-szm-st="" data-szm-cp="" data-szm-co="" data-ga-property="" onload="var lb_frame = this; this.contentWindow.postMessage({type: 'analytics', payload: JSON.stringify(lb_frame.dataset)}, lb_frame.src);" scrolling=no frameborder=0 width="100%" height="600" style="overflow: hidden; border: 0; min-width: 320px;"><p>Ihr Browser unterstützt keine iframes.</p></iframe>_
_scrolling=no_ 
eingesetzt (die letzte Version hier in en Kommentaren).
zu dem Link habe ich keinen Zugang. ggf. müsstet ihr mir da noch den Zugang zum Projekt einrichten, damit ich da ran komme.
